src/components/HuntingMapTutorial/HuntingMapTutorialMarkers.tsx

```diff
@@ -116,7 +116,7 @@ export const HuntingMapTutorialMarkers = () => (
 
     <p>
       The number of tracking markers that can be placed on the map at once is
-      limited to <Emphasis>100</Emphasis>. Once the 101st marker is placed, the
+      limited to <Emphasis>1000</Emphasis>. Once an additional marker is placed, the
```

Comment:
I meant using the variable here to make sure that documentation always reflects the actual count:

```suggestion
      limited to <Emphasis>{maxTrackingMarkerCount}</Emphasis>. Once an additional marker is placed, the
```

Just need to add the import at the top too.

Reply:
Adjusted.

Reply:
You need to import the variable at the top of the file 

```ts
import { maxTrackingMarkerCount } from 'config/markers';
```